packages/jmix-react-core/src/app/MainStore.ts

```diff
@@ -21,6 +21,7 @@ export class MainStore {
 
   static NAME = 'mainStore';
   static TOKEN_STORAGE_KEY = "jmixRestAccessToken";
+  static LOCALE_STORAGE_KEY = 'jmixLocale';
```

Comment:
Probably we will need to add some appName prefix otherwise consecutive usage of jmix frontends may share the same locale.